README.md

```diff
@@ -36,6 +36,7 @@ To get a local copy up and running follow these simple exampl
 - Go to the Project `cd "Math-magicians`
 - Run `npm install` to install the _dependencies_
 - Run `npm start` to run the live server.
+- Run `npm test` to run the tests.
 
 ## About me
 
```

Comment:
- [Optional] You haven't still fixed the License link in thIs README file. This link is still broken. It will be nice to fix it and render your README file professional.